src/commands/basemaps-mapsheet/create-mapsheet.ts

```diff
@@ -95,7 +95,9 @@ export const basemapsCreateMapSheet = command({
     const outputs = await createMapSheet(aerial, mem, rest, include, exclude);
 
     logger.info({ outputPath }, 'MapSheet:WriteOutput');
-    fsa.write(outputPath, JSON.stringify(outputs, null, 2));
+    const outputWritePromise = fsa.write(outputPath, JSON.stringify(outputs, null, 2));
+
+    await Promise.all([featuresWritePromise, outputWritePromise]);
```

Comment:
recommend returning Promise.all, no need to await in the method unless you need it's response

Reply:
Nothing in the caller `await`s for these results though. The other commands all seem to `await` their results before their end.

Reply:
agreed.  The method is async, which implies it returns a promise.  The other awaited results seem to be used